searvey/ioc.py

```diff
@@ -162,6 +212,18 @@ def normalize_ioc_stations(df: pd.DataFrame) -> gpd.GeoDataFrame:
     )
     return gdf
 
+def normalize_ioc_stations_api(df: pd.DataFrame) -> gpd.GeoDataFrame:
+    df = df.assign(
+        # fmt: off
```

Comment:
The `fmt` comments here seem to be redundant. We do use them in lines 202-207 but there we are exceeding the maximum line length and we want to suppress the error in `black`.